src/pages/Add.js

```diff
@@ -70,21 +73,9 @@ function Add() {
       votes: []
     };
 
-    const response = await fetch(
-      process.env.REACT_APP_POLLS_API ||
-        "https://my-json-server.typicode.com/meloneminze/umfragy/polls",
-      {
-        method: "POST",
-        headers: {
-          "Content-Type": "application/json"
-        },
-        body: JSON.stringify(poll)
-      }
-    );
-    const createdPoll = await response.json();
-    alert(`Created poll with the id ${createdPoll.id}`);
+    const createdPoll = await postPoll(poll);
+    history.push(`/polls/${createdPoll.id}/vote`);
   }
-
   return (
     <DIV>
```

Comment:
It does not affect the functionality but <DIV> should be <div> so we know it is a html tag